science/flrig/Portfile

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                flrig
+version             1.3.43
+categories          science
+platforms           darwin macosx
+license             GPL-3
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
+
+description         FLRIG is a transceiver control program
+long_description    FLRIG is a transceiver control program designed to be \
+    used either stand alone or as an adjunct to FLDIGI. The supported \
+    transceivers all have some degree of CAT. The FLRIG user interface \
+    changes to accommodate the degree of CAT support available for the \
+    transceiver in use.
+homepage            http://www.w1hkj.com
+master_sites        http://www.w1hkj.com/files/${name}/
+
+checksums           rmd160  38a96670d6278405c6d4adf01c33f3f0bc620cd8 \
+                    sha256  61c29f71e44a417f7a6b0ec35d465a585acda6c57dacddff83edfc25b73e24c7 \
+                    size    792283
+
+depends_build-append \
+    port:pkgconfig
+
+depends_lib-append \
+    port:fltk \
```

Comment:
Same question as above, can fltk-devel work here? Considering changing this to:

`path:lib/libfltk.dylib:fltk
`